packages/waku/src/lib/builder/serve-cloudflare.ts

```diff
@@ -1,41 +1,75 @@
 import { Hono } from 'hono';
+import { unstable_getPlatformObject } from 'waku/server';
 import { runner } from '../hono/runner.js';
+import type {
+  ExportedHandler,
+  fetch,
+  Request as CloudflareRequest,
+  Response as CloudflareResponse,
+} from '@cloudflare/workers-types/experimental';
 
 const loadEntries = () => import(import.meta.env.WAKU_ENTRIES_FILE!);
 let serveWaku: ReturnType<typeof runner> | undefined;
 
 export interface CloudflareEnv {
   ASSETS: {
-    fetch: (input: RequestInit | URL, init?: RequestInit) => Promise<Response>;
+    fetch: typeof fetch;
   };
 }
 
 export const app = new Hono<{
   Bindings: CloudflareEnv & { [k: string]: unknown };
 }>();
-app.use('*', (c, next) => serveWaku!(c, next));
+app.use('*', (c, next) => {
+  if (!serveWaku) {
+    throw new Error('serveWaku is not initialized');
+  }
+  const platform = unstable_getPlatformObject();
+  platform.honoContext = c;
```

Comment:
This is not good.
The platform object is something stable all the time. It shouldn't change for each request.
`unstable_getCustomContext` is for each request. And, the hono context is already exposed. #852